src/ImmersedBoundaries/immersed_boundary_grid.jl

```diff
@@ -131,3 +131,11 @@ function on_architecture(arch, ibg::IBG)
 end
 
 isrectilinear(ibg::IBG) = isrectilinear(ibg.underlying_grid)
+
+function Base.:(==)(grid1::IBG, grid2::IBG)
+    #check if grids are of the same type
```

Comment:
I don't understand the comment; the grids are of the same type if this method is triggered because both grids need to be `IBG`, right?